app/src/lib/editors/launch.ts

```diff
@@ -22,7 +22,7 @@ export async function launchExternalEditor(
     )
   }
 
-  if (__WIN32__ || __DARWIN__) {
+  if (__WIN32__ || __DARWIN__ || __LINUX__) {
```

Comment:
Not sure if it makes sense to keep this if statement

Reply:
It looks like you can remove it 🔥